commands/health.nicole

```diff
@@ -110,6 +110,7 @@ function cmd_logs_clear {
               xyz.openbmc_project.Collection.DeleteAll DeleteAll
 
   journalctl --rotate --vacuum-time=1s
+  log_audit "All logs are cleared"
```

Comment:
Let's change to `Cleared all logs` to indicate that it was done by this user, not on its own or by someone else